destination/models.go

```diff
@@ -0,0 +1,28 @@
+package destination
```

Comment:
Could you run `go fmt` just to be consistent on formatting and indentation.

Reply:
Done